repository-gcs/src/main/java/io/aiven/elasticsearch/repositories/gcs/GcsSettingsProvider.java

```diff
@@ -73,6 +73,13 @@ private Storage createGcsClient(final GcsStorageSettings gcsStorageSettings) {
                                 .setReadTimeout(gcsStorageSettings.readTimeout())
                                 .build())
                 .setHeaderProvider(() -> Map.of(HttpHeaders.USER_AGENT, HTTP_USER_AGENT))
+                .setRetrySettings(
+                        // Create retry settings from default by modifying the max attempts settings
+                        StorageOptions
+                            .getDefaultRetrySettings()
```

Comment:
Here are the defaults:

```
    return RetrySettings.newBuilder()
        .setMaxAttempts(6)
        .setInitialRetryDelay(Duration.ofMillis(1000L))
        .setMaxRetryDelay(Duration.ofMillis(32_000L))
        .setRetryDelayMultiplier(2.0)
        .setTotalTimeout(Duration.ofMillis(50_000L))
        .setInitialRpcTimeout(Duration.ofMillis(50_000L))
        .setRpcTimeoutMultiplier(1.0)
        .setMaxRpcTimeout(Duration.ofMillis(50_000L));
```

Do we see a value in providing the configuration settings for all / some of them, along with `max_retries`?